cmd/helm/reset.go

```diff
@@ -114,7 +114,7 @@ func (d *resetCmd) run() error {
 		}
 	}
 
-	fmt.Fprintln(d.out, "Tiller (the helm server side component) has been uninstalled from your Kubernetes Cluster.")
+	fmt.Fprintln(d.out, "Tiller (the helm server-side component) has been uninstalled from your Kubernetes Cluster.")
```

Comment:
- [ ] While we're still pursuing these corrections, why is "helm" written in lowercase here?